SparseGrids/tasgridExternalTests.cpp

```diff
@@ -1311,6 +1311,28 @@ bool ExternalTester::testAllRefinement() const{
         if (!testAnisotropicRefinement(f, &grid, type_ipcurved, 1, np, errs, 7)){
             cout << "ERROR: failed anisotropic refinement using leja ipcurved nodes for " << f->getDescription() << endl;  pass2 = false;
         }
+    }{
+        const BaseFunction *f = &f21c1c2periodic;
+        grid.makeFourierGrid(f->getNumInputs(), f->getNumOutputs(), 3, type_hyperbolic);
+        double transform_a[2] = {-1.0, -1.0};
+        double transform_b[2] = { 1.0,  1.0};
+        grid.setDomainTransform(transform_a, transform_b);
+        int np[5] = { 17, 35, 111, 273, 759 };
+        double errs[5] = { 1.28e-2, 2.80e-3, 1.97e-4, 6.78e-5, 5.65e-5 };
```

Comment:
Did you test the numbers with:
```
./gridtest refinement random
```
Normally the random seed for the tests is fixed and the results are deterministic, but the result does depend on the seed. Moving between compilers can result in different sequence of random test points which leads to fluctuations in the error.

Reply:
I ran it several times just now, and the tests passed each time. I got the error values from taking the max over 20,000 uniformly distributed points in MATLAB.

Reply:
OK, I wanted to make sure you didn't just run one deterministic test.